docs/source/index.md

```diff
@@ -1,9 +1,41 @@
-# Jupyter Project Documentation
+# Project Jupyter Documentation
 
-**Welcome to the Jupyter Project documentation**. This website acts as "meta" documentation for the Jupyter ecosystem.
-It has a collection of resources to navigate the tools and communities in this ecosystem, and to help you get started.
+Welcome to the Project Jupyter documentation site. Jupyter is a large umbrella
+project that covers many different software offerings and tools, including the
+popular [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/)
+and [JuyterLab](https://jupyterlab.readthedocs.io/en/stable/) web-based
+notebook authoring and editing applications. The Jupyter project and its
+subprojects all center around providing tools for interactive computing with
```

Comment:
> providing tools

I think that there are members of the community which would disagree and say that the focus of Project Jupyter is equally on **standards** and **tools** (which provide a reference implementation for standards)

Reply:
This is done